### DISCUSSION TITLE
Problem upgrading from 6.x to ~7.1.0-rc4

### DISCUSSION THREAD

**Discussion by lucrus73** [angular/angularfire#2967] [Q&A] [answered]
I'm trying to upgrade my app dependencies and my code, because I need [this commit](https://github.com/angular/angularfire/pull/2960/commits/4ce72ff1dc89ae86d56065b4a7281f2894d63868), so I edited my package.json and specified:

```
    "@angular/fire": "~7.1.0-rc.4",
    "firebase": "^9.0.0",

```
then I changed my imports from:

`import { AngularFirestore } from '@angular/fire';`

to

`import { AngularFirestore } from '@angular/fire/compat';`

My app is a Ionic5/Angular12 app, so I assume it uses Ivy by default. Looking [here](https://github.com/angular/angularfire/blob/master/docs/version-7-upgrade.md) seems to me I've done everything that was needed to upgrade.

However I get the following compilation error:

```
    Module '"@angular/fire/compat"' has no exported member 'AngularFirestore'.ts(2305)
    import AngularFirestore
```

What am I missing?

**Comment by jamesdaniels** [comment, marked as answer]
Import from `@angular/fire/compat/firestore`

**Reply by lucrus73** [reply]
Thanks, it works. Shouldn't it be documented [here](https://github.com/angular/angularfire/blob/master/docs/version-7-upgrade.md) too?
